Dockerfile

```diff
@@ -117,8 +117,15 @@ RUN --mount=type=cache,target=/var/cache/apt --mount=type=cache,target=/var/lib/
 
 COPY --from=build /usr/local/src/ort/scripts/*.sh /opt/ort/bin/
 
+# This can be set to a folder containing crt-Files for custom certificates that ORT and all build tools should know about.
```

Comment:
Prefer "directory" over folder" (also in the commit message); the latter is so Windows-ish 😉

Reply:
s/crt-Files/CRT-files/